services/supervisor/src/proto_structures.rs

```diff
@@ -124,6 +137,8 @@ pub enum ConfigChange {
     StartService(StartService),
     /// Existing service instance stop request.
     StopService(StopService),
+    /// Previously stopped service instance resume request.
```

Comment:
Nit: IMO, "... request" descriptions are weirdly phrased. I'd suggest something like "Request to resume a previously stopped service instance."